core/client/client_ip.go

```diff
@@ -186,6 +212,21 @@ func (c *IPClient) measureClockOffsetIP(ctx context.Context, log *zap.Logger, mt
 			return offset, weight, err
 		}
 
+		var ntsresp nts.NTSPacket
+		if c.Auth.Enabled {
+			cookies, responseID, err := nts.DecodePacket(&ntsresp, buf, KEData.S2cKey)
+			if err != nil {
+				log.Error("failed to authenticate packet", zap.Error(err))
+				return offset, weight, err
```

Comment:
```
if numRetries != maxNumRetries && deadlineIsSet && timebase.Now().Before(deadline) {
	log.Info("failed to decode and authenticate NTS packet", zap.Error(err))
	numRetries++
	continue
}
return offset, weight, err
```

Reply:
Btw.: Logging category "Error" means that a human operator should take a look (and possible also be alerted).

Reply:
Done and thank you for the hint